workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -89,6 +118,12 @@ export default class CostCenterService {
     return costCenter;
   }
 
+  private _mapDDBItemToCostCenter(item: { [key: string]: unknown }): CostCenter {
+    let costCenter: { [key: string]: unknown } = { ...item, accountId: item.dependency };
+    costCenter = removeDynamoDbKeys(costCenter);
```

Comment:
You don't need this function. Zod will take any item and parse out the desired object for you. You can do `CostCenterParser.parse({...item, accountId: item.dependency})` and get the same result. You may even be able to use a custom transformer on the parser to transform `item.dependency` into `item.accountId` on parse.

Reply:
Good call. Updated to not use `removeDynamoDbKeys`